taf/testlib/virtual_env.py

```diff
@@ -28,7 +28,9 @@
 #  "instance_type": "settings",
 #  "id": "993",
 #  "images_share_path": "/mnt/berta/oses/openstack",
-#  "mgmt_ip_cidr": "11.212.23.11/8",
+#  "external_net_ip_cidr": "11.212.23.0/24",
+#  "external_net_pool_start": "100",
+#  "external_net_pool_end": "199",
```

Comment:
```
+#  "external_net_ip_cidr": "11.212.23.0/24",
 +#  "external_net_pool_start": "100",
 +#  "external_net_pool_end": "199",
```
Since generating the env json is done by berta for our taf testing, I think this review also needs a change in berta, unless you can make this backwards compatible.